tests/sentry/integrations/discord/webhooks/test_message_component.py

```diff
@@ -23,11 +24,14 @@
     UNRESOLVED,
 )
 from sentry.models.release import Release
+from sentry.silo.base import SiloMode
 from sentry.testutils.cases import APITestCase
+from sentry.testutils.silo import assume_test_silo_mode, region_silo_test
 
 WEBHOOK_URL = "/extensions/discord/interactions/"
 
 
+@region_silo_test
```

Comment:
Should we have separate test case for the interactions that are handled on control silo?